### PR TITLE
add entity book

### PR DESCRIPTION
**Pull request by pachecowolox**
## Summary
- add properties entityScan in TrainingApplication
- change property value spring.datasource.url
## Trello Card
- https://trello.com/c/Hiu0adGP/14-crear-modelo-de-book

### CODE REVIEW

**Review by juanwongramirez — DISMISSED** [review]
Great!!

File: src/main/java/wolox/training/TrainingApplication.java [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
 @SpringBootApplication
+@EntityScan("wolox.training.models")
```
Comment (marcegl91) [inline src/main/java/wolox/training/TrainingApplication.java]: Why do you need this annotation?

File: src/main/java/wolox/training/models/Book.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private long id;
+
+	@Column(nullable = false)
```
Comment (marcegl91) [inline src/main/java/wolox/training/models/Book.java]: According to the card specification, genre can be nullable. The other fields should not be nullable, but this one can be set to null.

File: src/main/java/wolox/training/models/Book.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+	private String publisher;
+
+	@Column(nullable = false)
+	private String year;
+
+	@Column(nullable = false)
+	private Integer pages;
+
+	@Column(nullable = false)
+	private Integer isbn;
```
Comment (marcegl91) [inline src/main/java/wolox/training/models/Book.java]: ISBN should be a String, not an Integer.

**Review by marcegl91 — CHANGES_REQUESTED** [review]

**Review by marcegl91 — APPROVED** [review]

**Review by juanwongramirez — APPROVED** [review]

**Pull request merged**